app/controllers/concerns/recaptcha_concern.rb

```diff
@@ -0,0 +1,30 @@
+module RecaptchaConcern
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :allow_csp_recaptcha_src, if: :recaptcha_enabled?
+  end
+
+  private
+
+  RECAPTCHA_SCRIPT_SRC = [
+    'https://www.google.com/recaptcha/',
```

Comment:
Do we want these to be hardcoded like this, or would it be better to add a config?

Reply:
Personally, I don't see these needing to change very often, as they're pretty explicitly [documented in the reCAPTCHA guide](https://developers.google.com/recaptcha/docs/faq#im-using-content-security-policy-csp-on-my-website.-how-can-i-configure-it-to-work-with-recaptcha). That being said, this sounds similar to what was discussed in [an earlier comment about simulations](https://github.com/18F/identity-idp/pull/7761#discussion_r1095081717), so we might need to allow some configurability as part of LG-8833. Would you be alright to punt 'til then?

Reply:
yea i think i was thinking between lower env and prod, not to switch between different kinds of recaptcha.

Reply:
Ah, yeah, this won't add the CSP directives unless the feature is enabled, so we should be covered on that front 👍